prow/prowjobs/google/bms-toolkit/test-install-oracle.yaml

```diff
@@ -6,14 +6,11 @@ presubmits:
     decorate: true
     spec:
       containers:
-      - image: gcr.io/google.com/cloudsdktool/cloud-sdk:alpine
+      - image: quay.io/ansible/ansible-runner:latest
```

Comment:
From JC offline, he mentions that this is actually an official RedHat repo

Reply:
Links:
* [quay.io ](https://quay.io/) is Redhat's own container registry (https://access.redhat.com/articles/5925591).
* The Ansible org inside it is: https://quay.io/organization/ansible that provides the updates to the aforementioned  image.
* Ansible's official [dockerhub page](https://hub.docker.com/r/ansible/ansible-runner) reads: 
```
This repository is obsolete. The official Ansible Runner images now live at quay.io/ansible/ansible-runner
```